### DISCUSSION TITLE
Is it possible to store the editor state when the editor is closed?

### DISCUSSION THREAD

**Discussion by meisenbarth-work** [eclipse-theia/theia#12988] [Q&A] [answered]
VS Code stores the state (location, selection) of each file and restores it when the file is opened again. Theia does it for open editors when the application is closed. Is it possible to get the same behavior when the editor for the file is closed and opened again?

**Comment by vince-fugnitto** [comment, marked as answer]
@meisenbarth-work thank you for the discussion, I believe it used to work as intended: https://github.com/eclipse-theia/theia/issues/2432.
I'll open a regression instead (https://github.com/eclipse-theia/theia/issues/12989).
